crates/rust-analyzer/src/global_state.rs

```diff
@@ -63,6 +63,7 @@ pub(crate) struct GlobalState {
     pub(crate) last_reported_status: Option<lsp_ext::ServerStatusParams>,
     pub(crate) source_root_config: SourceRootConfig,
     pub(crate) proc_macro_clients: Vec<Result<ProcMacroServer, String>>,
+    pub(crate) overwritten_env_vars: Vec<(String, Option<OsString>)>,
```

Comment:
This is redundant, we already have `config` accessible here

Reply:
`overwritten_env_vars` is the original values of the environment variables before they are modified by `extraEnv`, so I don't believe that this redundant.

Reply:
ah well it should be once we no longer set the env processwide, that was my thought here sorry

Reply:
Removed